src/api.ts

```diff
@@ -267,3 +267,15 @@ export async function dedupeDependencies(
     `Deduplication is not supported for ${resolvedOptions.packageManager.name}`,
   );
 }
+
+export async function runScript(
+  name: string,
+  options: Pick<OperationOptions, "cwd" | "silent" | "packageManager"> = {},
+) {
+  const resolvedOptions = await resolveOperationOptions(options);
+
+  await executeCommand(resolvedOptions.packageManager.command, ["run", name], {
```

Comment:
I think for deno, it is `deno task` (not sure about bun).

Can you please make a recheck? (we might make a meta to map)